NOTICE.md

```diff
@@ -994,49 +637,41 @@ encodeurl - 1.0.2
  * License: MIT
  * Copyright: Copyright (c) 2016 Douglas Christopher Wilson
 
-encoding - 0.1.12
- * License: MIT
- * Copyright: Copyright (c) 2012-2014 Andris Reinman
-
 end-of-stream - 1.4.1
  * License: MIT
  * Copyright: Copyright (c) 2014 Mathias Buus
 
 engine.io - 3.3.2
  * License: MIT
- * Copyright: Copyright (c) 2014 Guillermo Rauch <guillermo@learnboost.com> Copyright (c) 2014 Guillermo Rauch <;guillermo@learnboost.com>;
 
-engine.io-client - 1.0.0
+engine.io - 3.4.0
  * License: MIT
+ * Copyright: Copyright (c) 2014 Guillermo Rauch <guillermo@learnboost.com> Copyright (c) 2014 Guillermo Rauch <;guillermo@learnboost.com>;
 
-engine.io-client - 3.3.1
+engine.io-client - 3.3.2
  * License: MIT
 
-engine.io-client - 3.3.2
+engine.io-client - 3.4.0
  * License: MIT
  * Copyright: Copyright (c) 2014-2015 Automattic <dev@cloudup.com> Copyright (c) 2014 Automattic  Inc. Copyright (c) 2012 Niklas von Hertzen
 
 engine.io-parser - 2.1.3
  * License: MIT
- * Copyright: Copyright (c) 2016 Guillermo Rauch (@rauchg)
-
-err-code - 1.1.2
- * License: MIT
 
-errno - 0.1.7
+engine.io-parser - 2.2.0
  * License: MIT
- * Copyright: Copyright (c) 2012-2015 [Rod Vagg](https
+ * Copyright: Copyright (c) 2016 Guillermo Rauch (@rauchg)
 
-es6-promise - 4.2.4
- * License: MIT
+errors - v0.8.1
+ * License: BSD-2-Clause
+ * Copyright: Copyright (c) 2015  Dave Cheney <dave@cheney.net>
 
 es6-promise - 4.2.6
  * License: MIT
- * Copyright: Copyright (c) 2014 Yehuda Katz  Tom Dale  Stefan Penner and contributors (Conversion to ES6 API by Jake Archibald) 
 
 es6-promise - 4.2.8
  * License: MIT
- * Copyright: Copyright (c) 2014 Yehuda Katz  Tom Dale  Stefan Penner and contributors (Conversion to ES6 API by Jake Archibald) 
+ * Copyright: Copyright (c) 2014 Yehuda Katz  Tom Dale  Stefan Penner and contributors (Conversion to ES6 API by Jake Archibald)
```

Comment:
OK...I'm beginning to think that these double spaces are intentional. 😉 As I see them also around "Tom Dale." 
It's my first time reviewing a file like this...so I'm unfamiliar with this kind of styling/formatting.